src/ConnectedMode/Binding/IUnintrusiveBindingController.cs

```diff
@@ -29,32 +30,57 @@ namespace SonarLint.VisualStudio.ConnectedMode.Binding
     internal interface IUnintrusiveBindingController
     {
         Task BindAsync(BoundSonarQubeProject project, IProgress<FixedStepsProgress> progress, CancellationToken token);
+        Task BindAsync(BoundServerProject project, IProgress<FixedStepsProgress> progress, CancellationToken token);
     }
 
     [Export(typeof(IUnintrusiveBindingController))]
     [PartCreationPolicy(CreationPolicy.NonShared)]
     internal class UnintrusiveBindingController : IUnintrusiveBindingController
```

Comment:
Why is this class called "Unintrusive"? Should we change the name? If not, maybe we should add a documentation to the interface to explain the name.

Reply:
Unintrusive = does not add anything into the csproj files or other git tracked files

Reply:
Please document that

Reply:
I don't think it makes sense to leave a comment to explain a pre-existing naming convention. This is not the only class that uses `Unintrusive` in it's name and It was not renamed in this PR. The word was introduced in version 7.0 to distinguish it from the Obsolete binding  (IObsoleteBindingConfigurationProvider).